src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -32,7 +32,7 @@
  * as defined in the
```

Comment:
On line 31, add "the" before "RSASSA-PSS".

Reply:
Ok.